index.html

```diff
@@ -793,6 +793,26 @@ <h2>Types</h2>
 specification that want to support extensibility in an interoperable fashion do.
         </p>
 
+        <div class="note">
+          <p>
+            It is important to note that the Verifiable Credentials data model is
+            intended to convey information about who said what about whom.
+            Credentials are not themselves an authority mechanism, but are
+            intended to convey information that can be useful in evaluating
+            whether to grant additional authority.
+          </p>
```

Comment:
VCs are absolutely "who said what about whom". (Well... they're not about the *question* "who said what about whom?", but about confirmation/verification that "A said B about C".)  In other words, VCs must convey sufficient information to permit a Receiver to Verify the Subject (C) and Issuer (A), and to understand the Statement (B).  *Possibly* the Receiver needs to be able to Verify that the Presenter is Permitted/Authorized to Present a VC of which they are not the Subject.

VCs let a border agent verify that a Passport -- a Credential which says CitizenOfNation (B) about Person (C) -- was Issued by the State Department (A) and is being Presented by that Person (C) -- via biometrics (height, weight, photo), embedded RFID, etc.  Whether that Person is granted any *authority* (e.g., permission to cross the border) by virtue of this Statement is not part of the VCs remit.

VCs let a police officer verify that a driver's license -- a Credential which says LicensedToOperateSimplePassengerVehicle (B) about Driver (C) -- was issued by the relevant Issuing Authority (A) and is being presented by that Driver (C) -- via biometrics (height, weight, photo), embedded RFID, etc.  Whether that Person is granted any *authority* (e.g., permission to drive *this* vehicle) by virtue of this Statement is not part of the VCs remit.

Current highly fallible verification is done using the characteristics of the presented document -- watermarks, holograms, overprinting, sample documents, etc.  VCs are supposed to be less fallible because of the ability to do something (as yet indeterminate, because that's the *protocol*) with the observable/dissectable characteristics (which may include partially encrypted and indecipherable content/payload) of the VC.

"Tittle-tattle or hearsay," as you call it, would be *un*Verifiable Credentials -- where I show the border agent a document which says something like "the UN said TallTed is a citizen of Earth, and may travel freely across all national borders" -- or perhaps I simply *say* this.  There is clearly no way to verify *these* credentials, because they convey nothing which might be used for such verification -- nor do they unambiguously identify me (Subject, and in this case, Holder).